components/smaily/actions/run-automation-workflow/run-automation-workflow.ts

```diff
@@ -0,0 +1,44 @@
+import { defineAction } from "@pipedream/types";
+import smaily from "../../app/smaily.app";
+
+export default defineAction({
+  name: "Run Automation Workflow",
+  version: "0.0.1",
+  key: "smaily-run-automation-workflow",
+  description: "Send Automation Workflow. [See docs here](https://smaily.com/help/api/automations-2/autoresponder)",
+  type: "action",
+  props: {
+    smaily,
+    automationWorkflowId: {
+      label: "Automation Workflow ID",
+      description: "The ID of the automation workflow. E.g. This URL `...sendsmaily.net/workflows/35` the ID will be `35`",
+      type: "string",
+    },
```

Comment:
Could the prop `automationWorkflowId` use async options with this endpoint? - https://smaily.com/help/api/automations-2/list-automation-workflows/